aa_stripe/management/commands/check_pending_webhooks.py

```diff
@@ -27,18 +29,22 @@ class Command(BaseCommand):
     help = "Check pending webhooks at Stripe API"
 
     def handle(self, *args, **options):
-        stripe.api_key = settings.STRIPE_API_KEY
-        pending_events_threshold = get_pending_events_threshold()
+        stripe.api_key = stripe_settings.API_KEY
+        pending_events_threshold = stripe_settings.PENDING_EVENTS_THRESHOLD
 
         pending_events = []
-        last_event = StripeWebhook.objects.latest("created").id
+        last_event = StripeWebhook.objects.last()
+        last_event_id = last_event.id if last_event else None
         while True:
-            event_list = stripe.Event.list(starting_after=last_event, limit=100)  # 100 is the maximum
-            pending_events += event_list["data"]
+            event_list = stripe.Event.list(ending_before=last_event_id, limit=100)  # 100 is the maximum
+            for event in event_list["data"]:
+                if event["pending_webhooks"] > 0:
```

Comment:
Please fix.
The story says: "count the amount of events and sent alerts depending on that. The thing is that those are the awaiting events." so clearly about counting not received events. You are checking amount of pending webhooks here for each event not pending webhooks. The list you are receiving here should contain only the events that we not have on our end, so simply counting those should be enough.